{
  "comments": [
    {
      "unresolved": false,
      "key": {
        "uuid": "f6524c2b_b32208dc",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 7
      },
      "lineNbr": 0,
      "author": {
        "id": 1000340
      },
      "writtenOn": "2022-02-24T19:10:01Z",
      "side": 1,
      "message": "CI+2 fails are due to things unrelated to this patch.",
      "revId": "4b1ac4793176810f6e741ba898d6583241a39ae9",
      "serverId": "8f6f209b-db1a-4cbf-aa44-c8bc30e9bfda"
    }
  ]
}